Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <iostream>
+
+#include <wil/result.h>
+#include <wil/cppwinrt.h>
+
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Microsoft.Windows.PushNotifications.h>
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt::Windows::Foundation;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory
```

Comment:
I'm nervous putting the entire link here because this link will change to https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory-**aad** once stable release is out. How about `// follow "Configure your app's identity in Azure Active Directory" at https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart`?

Also, how about of instead of "To obtain an AAD RemoteIdentifier for your app," we say "To obtain your Azure AppId," since there is 0 reference of a "RemoteIdentifier" in the quickstart?

Reply:
done